nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -74,8 +110,7 @@ def create_nncf_graph(model: torch.fx.GraphModule) -> PTNNCFGraph:
         nncf_graph = PTNNCFGraph()
 
         for source_node in model.graph.nodes:
-            node_type, node_metatype = GraphConverter._get_node_type_and_metatype(source_node)
-
+            node_type, node_metatype = GraphConverter._get_node_type_and_metatype(source_node, model)
             nncf_graph.add_nncf_node(
                 node_name=source_node.name,
                 node_type=node_type,
```

Comment:
@anzr299, please do something like this
```suggestion
            node_metatype = GraphConverter._map_fx_unique_metatypes(node_metatype)
            
            nncf_graph.add_nncf_node(
                node_name=source_node.name,
                node_type=node_type,
```

Reply:
Done!